ci.not.Rune.yml

```diff
@@ -0,0 +1,57 @@
+app-id: ci.not.Rune
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: rune
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --own-name=org.mpris.MediaPlayer2.rune_player.*
+  - --talk-name=org.kde.StatusNotifierWatcher
+rename-desktop-file: rune.desktop
+rename-appdata-file: rune.metainfo.xml
+rename-icon: rune
+modules:
+  - shared-modules/libayatana-appindicator/libayatana-appindicator-gtk3.json
+  - name: rune
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 assets/rune.desktop -t /app/share/applications/
+      - install -Dm644 assets/rune.metainfo.xml -t /app/share/metainfo/
+      - rm -fr assets/icons/1024x1024
+      - mkdir -p /app/share/icons
```

Comment:
```suggestion
      - mkdir -p /app/share/icons/hicolor
```

Reply:
but you are copying to `/app/share/icons/hicolor` without creating the hicolor directory first

Reply:
If I create `hicolor` first instead of letting `cp` to create it, I would need to use glob/find with -exec to copy the content of `icons`, and I prefer avoid that as things are being complicated by doing so, but if the guidelines are against doing this I can change that.

Reply:
> If I create hicolor first instead of letting cp to create it,

Ok so the next command is incorrect then?

```
cp -r assets/icons /app/share/icons/hicolor
```

should be `cp -r assets/icons /app/share/icons/` ?

If the `hicolor` exists inside `assets/icons`

Reply:
No, `assets/icons` only includes what's needed to be inclued to `hicolor`.

Reply:
so how is this working? cp cannot create directories for you

Reply:
cp can do that:

```
$ cd $(mktemp -d)
$ pwd
/tmp/tmp.NG2c2qIqSc
$ mkdir -p a/b/c
$ mkdir -p d/e/f
$ echo test > a/b/c/z
$ cp -r a/b/c d/e/f/g
$ cat d/e/f/g/z
test
```

Reply:
interesting, looks like it only works when used with `-r`, normally it fails.